simapp/app.go

```diff
@@ -36,6 +36,9 @@ import (
 	paramproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 	"github.com/cosmos/cosmos-sdk/x/slashing"
 	"github.com/cosmos/cosmos-sdk/x/staking"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
```

Comment:
Make sure these changes are formatted correctly. You can run goimports manually or, preferably, run `make fmt`

Reply:
Alternatively, just run:

```
git config core.hooksPath contrib/githooks
```

The `git` pre-commit hook will fix code formatting for you.

Reply:
Hi Alexander,
both `gofmt`, and `goimports` do not detect any formatting issue. I am using vscode with auto-formatting on save.  

In addition the command `make fmt` does not exist. There is a command `make format` in the make file which also does not detect any formatting issue.

Do you want me to remove the extra line in the import manually?

Reply:
Ahh, sorry, I meant `make format`. Tip, you can use `tab` to autocomplete.

WRT to formatting, the grouping is what I'm referring to. `make format` should take care of this for you.

Reply:
As I said abobe, after running `make format` nothing happens. The file `app.go` didn't change at all:

![Selection_064](https://user-images.githubusercontent.com/30392990/84440433-468dcc00-ac3a-11ea-9b71-bb857da391ac.png)

Reply:
Goimport isn't smart enough to recognize this case unfortunately. Could you just remove the space manually.